package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "compressed-size-action",
-  "version": "2.5.0",
+  "version": "3.0.0",
```

Comment:
Please revert this & the unrelated lock file change, we'll bump the version when releasing.